scripts/validate_packaged_v8_resources.py

```
@@ -0,0 +1,144 @@
+
+def _validate_inventory(package_root: Path, runtime_root: Path, *, label: str) -> None:
+    fallback_root = runtime_root / "Lib" / "schemas"
+    if fallback_root.exists() or fallback_root.is_symlink():
+        raise PackagedV8ResourceError(f"{label} runtime unexpectedly contains a Lib/schemas fallback tree")
+
+    for relative, expected_names in EXPECTED_V8_RESOURCES.items():
+        directory = package_root.joinpath(relative)
+        if directory.is_symlink() or not directory.is_dir():
+            raise PackagedV8ResourceError(
+                f"{label} DefenseClaw v8 resource directory is not a regular directory: {relative}"
+            )
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)